### DISCUSSION TITLE
一些小建议

### DISCUSSION THREAD

**Discussion by a546109781** [the1812/Bilibili-Evolved#3868] [功能建议 / Ideas]
按视频分区记忆倍速，主要针对音乐区 每次听歌都要手动切回1倍速
另外希望可以加一个视频音量调节条 上下拖动固定 +-5或+-10 或者拉高几倍 原版的老是对不齐整数让人血压飙升

**Comment by the1812** [comment]
想对齐就用键盘调, 固定+-10
